stylesheets/default_stylesheet.py

```diff
@@ -0,0 +1,81 @@
+from dash import Dash, html, Input, Output, dcc, ctx, State
+import dash_bootstrap_components as dbc
+import dash_cytoscape as cyto
+import module_data 
```

Comment:
I think `module_data` is unused here and this can be removed. Depending on what editor you're using there shoul be an extension that will remove unused imports automatically.